lib/buildEmbed.js

```diff
@@ -31,7 +46,9 @@ function liteEmbed(id, options, index) {
     out += liteOpt.js.inline ? `<script>${inlineJs}</script>\n` : `<script defer="defer" src="${liteOpt.js.path}"></script>\n`;
   }
   out += `<div id="${id}" class="${options.embedClass}">`;
-  out += `<lite-youtube videoid="${id}" style="background-image: url('https://i.ytimg.com/vi/${id}/hqdefault.jpg');">`;
+  out += `<lite-youtube videoid="${id}" style="background-image: url('https://i.ytimg.com/vi/${id}/hqdefault.jpg');"`
+  out += params ? ` params="` + params + `"`: "";
```

Comment:
I think it's easier to read this when using template literals!

```suggestion
  out += params ? ` params=${params}`: "";
```

Reply:
even better :)